src/js/blight/BlightContainer.jsx

```diff
@@ -25,19 +24,26 @@ export default class BlightContainer extends Component {
   handleRemoveBlight(e) {
     const newBlight = this.state.numBlight - 1;
     this.setState({ numBlight: newBlight});
-    if(newBlight <= 0 && this.state.endGameTrigger){
-      this.setState({ blightCard: { "desc": "", "name": "GAME OVER", "newblights": 0 } });
+    if(newBlight <= 0 && this.state.endGameTrigger === true){
       this.setState({ numBlight: 0 });
+      this.setState({ blightCard: { "desc": "", "name": "GAME OVER", "newblights": 0 } });
     }
     else if(newBlight === 0 ){
-      this.setState({ endGameTrigger: true });
       let card = _.shuffle(blightCards)[0];
-      this.setState({ blightCard: card });
-      this.setState({ numBlight: card.newBlights*this.props.players });
+      this.setState({ numBlight: card.newBlights*this.props.players,
+                      blightCard: card,
+                      endGameTrigger: true 
+      });
     }
   }
   
   render() {
+    // Help idk y this doesnt work
+    //if(this.endGameTrigger === true){
```

Comment:
`this.endGameTrigger` doesn't exist... Did you mean `this.state.endGameTrigger`? 
If you want to debug, try putting `debugger;` on its own line and open the console on your browser.